src/core/radio/radio_platform.cpp

```diff
@@ -145,3 +145,10 @@ OT_TOOL_WEAK void otPlatRadioSetMacFrameCounter(otInstance *aInstance, uint32_t
     OT_UNUSED_VARIABLE(aInstance);
     OT_UNUSED_VARIABLE(aMacFrameCounter);
 }
+
+OT_TOOL_WEAK uint64_t otPlatRadioGetNow(otInstance *aInstance)
+{
+    OT_UNUSED_VARIABLE(aInstance);
+
```

Comment:
return otPlatTimeGet()?

Reply:
Not all platform supports this API. Currently, only nrf528xx and simulation supports this. Others support `otPlatAlarmMilliGetNow`. So need to return a special value here as a weak implementation.